pnpm-lock.yaml

```diff
@@ -8987,6 +9123,14 @@ packages:
     dev: true
     optional: true
 
+  /@rspack/binding-linux-arm64-gnu@0.3.14:
```

Comment:
Why is Rspack 0.3.14 added to lock file

Reply:
good points, forget to update vue-template version